site/en/guides/border-router/espressif-esp32.md

```diff
@@ -23,14 +23,18 @@ To set up the environment, please follow the [official installation guide](https
 Clone the [esp-idf](https://github.com/espressif/esp-idf) and the [esp-thread-br](https://github.com/espressif/esp-thread-br) repository.
 
 ```
-$ git clone -b v5.1.1 --recursive https://github.com/espressif/esp-idf.git
+$ git clone -b v5.1.2 --recursive https://github.com/espressif/esp-idf.git
 $ cd esp-idf
 $ ./install.sh
 $ . ./export.sh
 $ cd ..
-$ git clone --recursive https://github.com/espressif/esp-thread-br.git
+$ git clone -b v1.0 --recursive https://github.com/espressif/esp-thread-br.git
 ```
 
+> aside positive
+>
```

Comment:
Please remove these two lines, you only need the one starting with `> Note:`

"aside positive" is only used in the codelabs format

Reply:
Done.